remote/inc/patterns/posts-grid.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Posts grid.
+ *
+ * @package Remote
+ */
+
+return array(
+	'title'      => __( 'Posts grid', 'remote' ),
+	'categories' => array( 'pages' ),
+	'blockTypes' => array( 'core/query' ),
+	'content'    => '<!-- wp:group {"align":"full","layout":{"inherit":true}} -->
+	<div class="wp-block-group alignfull"><!-- wp:query {"queryId":0,"query":{"perPage":6,"pages":0,"offset":0,"postType":"post","order":"desc","orderBy":"date","author":"","search":"","exclude":[],"sticky":"","inherit":true,"taxQuery":null},"tagName":"main","displayLayout":{"type":"flex","columns":2},"align":"wide"} -->
+	<main class="wp-block-query alignwide"><!-- wp:post-template -->
+	<!-- wp:separator {"className":"is-style-wide"} -->
+	<hr class="wp-block-separator is-style-wide"/>
+	<!-- /wp:separator -->
+	
+	<!-- wp:post-featured-image {"width":"100%","height":"220px","style":{"spacing":{"margin":{"top":"60px","bottom":"30px"}}}} /-->
+	
+	<!-- wp:post-date {"format":"M j","style":{"typography":{"textTransform":"uppercase"}},"fontSize":"small"} /-->
```

Comment:
I think post-date is always uppercase in this theme. Should we move this to theme.json?